x-pack/plugins/index_management/public/application/components/mappings_editor/components/document_fields/fields/field_types/boolean_type.tsx

```diff
@@ -24,8 +25,9 @@ const getDefaultToggleValue = (param: string, field: FieldType) => {
     case 'boost': {
       return field[param] !== undefined && field[param] !== getFieldConfig(param).defaultValue;
     }
+    case 'meta':
```

Comment:
I think "null_value" could have been left, and simply add "meta" on top of "boost"